src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
```

Comment:
Functions traditionally do things, their names represent actions. So we tend to start them with verbs. "color" is rather ambiguous and can be perceived as a noun too. This is why I'd recommend maybe changing it to something like:
```suggestion
def colorize_menu(
    column_number: int,  # pylint: disable=unused-argument
    column_name: str,  # pylint: disable=unused-argument
    menu_entry: Dict[str, Any],
) -> Tuple[int, int]:
```
As for the argument names, I'd try to avoid shortening the vars. This tradition of trying to fit the names in a few characters is coming from ancient languages that used to have actual limitations for var names forcing people to come up with obscure abbreviations. But in modern tech, especially in Python that values readability and writing code for humans to read, there's no need to do this. In fact, it's quite welcome to use words where they can contribute the context of what the vars mean on the current level of abstraction.
So I suggest doing exactly that.

Reply:
This pattern (function name, signature) is used across all actions.

```
src/ansible_navigator/actions/inventory.py:31:def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
src/ansible_navigator/actions/inventory.py:269:                    color_menu_item=color_menu,
src/ansible_navigator/actions/config.py:33:def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
src/ansible_navigator/actions/config.py:177:                    color_menu_item=color_menu,
src/ansible_navigator/actions/collections.py:32:def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
src/ansible_navigator/actions/collections.py:170:                    color_menu_item=color_menu,
src/ansible_navigator/actions/images.py:67:    def color_menu(self, colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
src/ansible_navigator/actions/images.py:188:                    color_menu_item=self.color_menu,
src/ansible_navigator/actions/run.py:58:def color_menu(_colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
src/ansible_navigator/actions/run.py:541:                    color_menu_item=color_menu,
```

I agree it can be improved, but that should be a follow on PR across all actions, probably to include the use of named arguments in the caller.

Reply:
Okay. @alisonlhart feel free to click `Resolve conversation` once you read this.